### ISSUE TITLE
Kopernicus Mod support - galaxies not visible when using TST with this mod

### ISSUE THREAD

**Issue by JPLRepo** [JPLRepo/TarsierSpaceTechnology#5]
Originally Posted by Rhedd 
I love this mod, but unfortunately the galaxies aren't visible if you're using the latest Kopernicus, so no galaxy viewing if you have the Outer Planets mod, for example. Is that possibly fixable?
Originally Posted by miguel11 
I can also confirm this. Kopernicus/Outer Planets Mod are very popular mods. IMHO, I would make TST compatible with some of the most popular planets pack because after all, this mod is about watching planets, and other stuff in the universe, Right.

Cheers...

**Comment by JPLRepo** [comment]
It appears this is because Kopernicus completely replaces the universe/sun/planets generation in KSP. With some very very COOL code.
And TST does not expect this. So whilst this isn't a bug, it's an enhancement request to support what Kopernicus does.
So TST - you can see all the planets, but you won't see the galaxies because TST ADDS them itself in reference to the SUN. Which in Kopernicus doesn't exist any more.. at least not in the same form within the code.
So I need to change TST to support Koernicus by knowing when it is installed alongside it and change the reference point for adding the galaxies.

**Comment by JPLRepo** [comment]
The previous comment was wrong. It does not appear to be the reference point. This is proving problematic. Need to dive deep into the kopernicus code to figure out what it is changing.
You can see all planets and take pictures of them when using Kopernicus. 
The galaxies ARE there, you can target them and you can take pictures/collect science.
The issue is the galaxie texture/picture is not visible in the camera window.

**Comment by JPLRepo** [comment]
Finally!! figured it out... Ok so it WAS the very first thing I posted...
PLUS - the version of Kopernicus is still very much a WIP and the version I was testing with (distributed with the KerbolPlus mod? collection of configs) was BROKEN.
When I downloaded the latest Kopernicus and replaced the one that came with KerbolPlus - TA-DA!!! about 6 hours of my life wasted trying to figure this out for nothing. That will teach me.
So this is now fixed as-in if you use the latest Kopernicus and my about to be released TST it works perfectly.
